src/main/webapp/script.js

```diff
@@ -70,11 +69,33 @@ function loadGrocery() {
 
 function createGroceryElement(grocery){
     const groceryElement = document.createElement('li');
-        groceryElement.className = 'task';
+    groceryElement.className = 'task';
 
+    // changes string depending on if a member was assigned an email. 
```

Comment:
can you clarify this comment? which string is changed? And what does it mean for a member to be asigned an email?

Reply:
If assigned email is empty then only show the item else show the item and the assigned email. The string being changed is the item in the JSON list.